odoo_sh/advanced/frequent_technical_questions.rst

```diff
@@ -6,6 +6,16 @@
 Frequent Technical Questions
 ==================================
 
+"Automated actions do not run at the exact time they were expected"
+-------------------------------------------------------------------
+
+On the Odoo.sh platform, we cannot guarantee an exact running time for automated actions.
+
+This is due to the fact that there might be multiple customers on the same server, and we must guarantee a fair share of the server for every customer. Automated actions are therefore implemented slightly differently than on a regular Odoo server, and are run on a *best effort* policy.
+
```

Comment:
we might take the opportunity to add a short message explaining best practice for crons such as batches explicitly commited written in an idempotent way (maybe it's already present somewhere else in the docs in such case a link is enouf)

Reply:
I'll add it, good idea!